doc/source/tests/pytest_framework.rst

```diff
@@ -86,18 +124,12 @@ First, check out the highest numbered gold standard tag and compile ``enzo-e``.
    $ git checkout gold-standard-1
```

Comment:
I know this is not changed in this PR, but aren't we on gold-standard-004?

Reply:
You are correct. I have updated this accordingly (and I've added a note commenting that readers will need to update the number in the future)